test/utils/image/manifest.go

```diff
@@ -68,15 +68,15 @@ func (i *Config) SetVersion(version string) {
 	i.version = version
 }
 
-func Init(repoList string) {
-	registry, imageConfigs, originalImageConfigs = readRepoList(repoList)
+func Init(repoList, imageRepository string) {
```

Comment:
This is a bit complex to follow, so once you get the flags values this function is called with the value of the flags